src/Community/clique_percolation.jl

```diff
@@ -42,13 +42,15 @@ CliquePercolation(;k=3) = CliquePercolation(k)
   kcliques = filter(x->length(x)>=alg.k, maximal_cliques(g))
   nc = length(kcliques)
   # graph with nodes represent k-cliques
-  h = Graph(nc)
+  h = SimpleGraph(nc)
```

Comment:
Again, where we have `Graph` in the LG code, we want to change it to `SimpleGraph`. (Same for DiGraph and Edge).